src/collections/raw_vec.rs

```diff
@@ -56,6 +56,10 @@ pub struct RawVec<'a, T> {
     a: &'a Bump,
 }
 
+/// `RawVec<T>` in `Sync` if `T` is.
+/// This is valid as `RawVec` doesn't access inner `Bump` in `&self` methods.
+unsafe impl<'a, T> Sync for RawVec<'a, T> where T: Sync {}
```

Comment:
Sorry, I forgot to remove this code, when I was trying to make `Vec<T>: Sync where T: Sync` and found it is in fact unsound, because clonned `Vec<T>` may access `Bump`.
I explained it here #72 as well.

Reply:
Ah, thanks I didn't see the code example in that issue originally, and now that I've seen it, it's very clear to me.